packages/home-spa/README.md

```diff
@@ -14,14 +13,19 @@ npm install
 ```
 
 ## Usage
+
 ### Development server
+
 - Change .env.example to .env
 - Add URL to API_URL in .env file
-```
+
+```sh
 API_URL=https://example.com/api
 ```
+
 - Schema
-```
+
+```sh
```

Comment:
It's not a shell command. The syntax is more like a JS object, so it should be `js`

```suggestion
```js
```